src/lib.rs

```diff
@@ -391,6 +391,11 @@ doc_comment! {
     include_str!("../README.md")
 }
 
-/// Macro which link [`native_model`](https://crates.io/crates/native_model) to the Native DB. See [`Builder.define`](struct.Builder.html#method.define) for more information.
+/// Macro which link [`native_model`](https://crates.io/crates/native_model) to the Native DB. See
+/// [`Builder.define`](struct.Builder.html#method.define) for more information.
+///
+/// Optional parameter `export_keys` controls whether the model keys enum is visible outside of the
+/// crate, default value is false with visibility limited to `pub(crate)`. Use
+/// `#[native_db(export_keys = true)]` to make enum visible outside of crate (ie. `pub`).
```

Comment:
What do you think about moving the documentation of `export_keys` into the documentation of [`define`](https://github.com/Grimeh/native_db/blob/f3523f9ae2c2158ff918e6bdc71173726fd8f06d/src/models.rs#L69-L304) in order to have all the model creation documentation in one place?